.github/workflows/e2e_ci.yml

```diff
@@ -167,7 +167,7 @@ jobs:
           UBSAN_OPTIONS: print_stacktrace=1
         run: |
           cd build/test/libsinsp_e2e/
-          sudo -E ./libsinsp_e2e_tests ${{ matrix.driver.option }}
+          sudo -E ./libsinsp_e2e_tests ${{ matrix.driver.option }} --gtest_filter=-sys_call_test.forking_clone_fs
```

Comment:
Can we disable it directly from the test suite (`DISABLED_...`) instead of just the CI with an explanation about why it is disabled?